autopush/router/simple.py

```diff
@@ -153,9 +155,11 @@ def route_notification(self, notification, uaid_data):
             returnValue(self.stored_response(notification))
         try:
             result = yield self._send_notification_check(uaid, node_id)
-        except (ConnectError, UserError, ConnectionRefusedError):
+        except (ConnectError, UserError, ConnectionRefusedError), exp:
```

Comment:
nit: as exp here as well
double-nit: its fairly universal python convention to abbreviate exception to exc